### PR TITLE
feat: hide vault address when not ready

### PR DESCRIPTION
**Pull request by peronczyk**

### CODE REVIEW

**Comment by Liubov-crypto** [comment]
I can copy address of vault right after his creation, the page is available: 

https://github.com/superhero-com/superhero-wallet/assets/69896204/6fb9034c-2605-444b-b216-9decde6ade74

File: src/popup/components/AccountCardBase.vue [hunk 1]
```diff
@@ -15,30 +19,54 @@
...
-  </div>
+  </BtnBase>
 </template>
 
 <script lang="ts">
-import { defineComponent } from 'vue';
+import { PropType, computed, defineComponent } from 'vue';
+import { RouteLocationNamedRaw } from 'vue-router';
+
+import { IS_MOBILE_APP } from '@/constants';
```
Comment (CedrikNikita) [inline src/popup/components/AccountCardBase.vue:29]: This variable is not used in this file.

File: src/popup/components/AccountCardBase.vue [hunk 2]
```diff
@@ -15,30 +19,54 @@
...
 
 export default defineComponent({
+  components: {
+    BtnBase,
+  },
   props: {
+    idx: { type: Number, default: -1 },
+    address: { type: String, default: '' },
+    to: { type: Object as PropType<RouteLocationNamedRaw>, default: null },
+    hideNext: Boolean,
```
Comment (CedrikNikita) [inline src/popup/components/AccountCardBase.vue:42]: This prop is not used in this file.

File: src/popup/components/AccountCardBase.vue [hunk 3]
```diff
@@ -15,30 +19,54 @@
...
+import { getAddressColor } from '@/utils';
+
+import BtnBase from './buttons/BtnBase.vue';
 
 export default defineComponent({
+  components: {
+    BtnBase,
+  },
   props: {
+    idx: { type: Number, default: -1 },
```
Comment (CedrikNikita) [inline src/popup/components/AccountCardBase.vue:39]: This prop is not used in this file.

**Review by CedrikNikita — APPROVED** [review]

**Comment by Liubov-crypto** [comment]
LGTM

**Pull request merged**